sdk/communication/Shared/src/HMACAuthenticationPolicy.cs

```diff
@@ -77,18 +78,19 @@ private void AddHeaders(HttpMessage message, string contentHash)
 
             message.Request.Headers.SetValue("x-ms-content-sha256", contentHash);
             message.Request.Headers.SetValue(HttpHeader.Names.Date, utcNowString);
```

Comment:
Do we still need to add the `Date` header?

Reply:
We don't have to. But keeping a copy just to see two dates in service log can be helpful to see if Date gets modified anywhere along the way

Reply:
I do not think that is necessary anymore. The `Date` header is not being used by our service when the `x-ms-date` is present and our new standard that you were updating the doc for does not need the `Date` header. I think now that we have the issue fixed, we should align with the standard that we are documenting.